test/parallel/test-tz-version.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasIntl) {
+  common.skip('missing intl');
+}
+
+const fixtures = require('../common/fixtures');
+
+// This test ensures the correctness of the automated timezone upgrade PRs.
+
+const { strictEqual } = require('assert');
+const { readFileSync } = require('fs');
+
+const expectedVersion = readFileSync(fixtures.path('tz-version.txt'), 'utf8').trim();
+strictEqual(process.versions.tz, expectedVersion);
```

Comment:
We support lots of different ways of building Node.js and ICU, including pointing to external ICU source/data files, so this test is going to need guards to work properly if its going to be part of the regular test suite. 

https://github.com/nodejs/node/blob/1af63a90ca3a59ca05b3a12ad7dbea04008db7d9/configure.py#L549-L590

Reply:
I have added a guard, hope that's enough?